applications/RANSApplication/custom_constitutive/rans_newtonian_3d_law.h

```diff
@@ -0,0 +1,135 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_RANS_NEWTONIAN_LAW_3D_H_INCLUDED)
+#define KRATOS_RANS_NEWTONIAN_LAW_3D_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_constitutive/newtonian_3d_law.h"
+
+namespace Kratos
+{
+/**
+ * @brief This class is extending Newtonian3DLaw in FluidDynamicsApplication
+ *
+ * This class is used to extend Newtonian3DLaw in FluidDynamicsApplication
+ * to include turbulent viscosity from Bossinesq Hypothesis.
+ *
+ */
+class KRATOS_API(RANS_APPLICATION) RansNewtonian3DLaw : public Newtonian3DLaw
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    using BaseType = Newtonian3DLaw;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * Counted pointer of RansNewtonian3DLaw
+     */
+
+    KRATOS_CLASS_POINTER_DEFINITION(RansNewtonian3DLaw);
+
+    /**
+     * Life Cycle
+     */
+
+    /**
+     * Default constructor.
+     */
+    RansNewtonian3DLaw();
+
+    /**
+     * Clone function (has to be implemented by any derived class)
+     * @return a pointer to a new instance of this constitutive law
+     */
+    ConstitutiveLaw::Pointer Clone() const override;
+
+    /**
+     * Copy constructor.
+     */
+    RansNewtonian3DLaw(const RansNewtonian3DLaw& rOther);
+
+    /**
+     * Destructor.
+     */
+    ~RansNewtonian3DLaw() override;
```

Comment:
```suggestion
    ~RansNewtonian3DLaw();
```

Reply:
Don't we need override here? (we have virtual method)

Reply:
This is not a standard method, is the destructor. https://github.com/isocpp/CppCoreGuidelines/issues/721

Reply:
done :)